src/crypto/qat/qcccrypto.cc

```diff
@@ -20,51 +26,111 @@ static std::ostream& _prefix(std::ostream* _dout)
 // -----------------------------------------------------------------------------
 
 /*
- * Poller thread & functions
-*/
+ * Callback function
+ */
+static void symDpCallback(CpaCySymDpOpData *pOpData,
+                        CpaStatus status,
+                        CpaBoolean verifyResult)
+{
+  if (nullptr != pOpData->pCallbackTag)
+  {
+    if (static_cast<QatCrypto*>(pOpData->pCallbackTag)->complete()) {
+      static_cast<QatCrypto*>(pOpData->pCallbackTag)->completion_handler(std::error_code{});
```

Comment:
shouldn't we pass `CpaStatus status` instead of success?

Reply:
> shouldn't we pass `CpaStatus status` instead of success?

Yes, I changed the Signature, `Signature = void(CpaStatus);`. And here pass `CPA_STATUS_SUCCESS`.